### PR TITLE
Template setup

### PR DESCRIPTION
**Pull request by DaniMori**
Creates and sets up the template of the YOS Awards call form (to be filled in with our responses to the fields).
Only the free text fields are versioned; the rest may be added later on.

### CODE REVIEW

**Comment by DaniMori** [comment]
Guys, review this pull request (PR), it only versions the empty form template. In your review please approve/comment/request changes, as you consider necessary.

**Review by robertomediavilla — DISMISSED** [review]

**Comment by DaniMori** [comment]
Sorry @robertomediavilla, I cleaned up the repository from unneeded folders and the `renv` environment manager (we won't be using R actually in this project). Please review & approve again.

**Review by robertomediavilla — APPROVED** [review]
Kick-off repo approval

**Review by Cristrinaranjus — APPROVED** [review]

**Review by Cristrinaranjus — APPROVED** [review]

**Comment by Cristrinaranjus** [comment]
I noticed that the branch where the changes are being made is called "feture" instead of "feature". How can a PR be made to modify that?

**Comment by DaniMori** [comment]
> I noticed that the branch where the changes are being made is called "feture" instead of "feature". How can a PR be made to modify that?

Hahah I'm so sorry... That cannot be modified with a PR; there is a (very obscure) way to modify it... so let's live with it :)

File: output/Form.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+<!--# blank field, max 400 words -->
+
+# Follow-up: How do you intend to use the award?
+
+*How would this award support you / your team in taking forward your current
+initiative, or would you like to initiate a new project with the financial
+resources from the award?*
+
+<!--# blank field, max 400 words -->
```
Comment (JesusSierralaya) [inline output/Form.md:51]: I think everything is fine. This is only a try to test the review.

**Review by JesusSierralaya — APPROVED** [review]
I approve the previous editions!

**Review by Leafrancia — APPROVED** [review]

**Comment by Leafrancia** [comment]
There are a lot of deleted files, is that normal?

**Comment by DaniMori** [comment]
Yes @Leafrancia , see specifically commits dc20079c9fd1530f49f07e23a191b78595834159 and f7a49202134caeb7262d2e6ffb9972285e017e47

**Pull request merged**